src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,330 @@
+to standardize the set of interactions between receiver and sender, specifically the process by which a receiver
+directs a sender to their digital wallet to complete a payment.
+We believe this protocol will enable a largely improved user experience in e-commerce transactions by reducing the number
+of steps a user must take to complete a transaction, creating a consistent and familiar checkout pattern, and lowering
+the barrier to entry use for cryptocurrency novices.
+
+The second priority of PayID Easy Checkout is to limit the engineering effort needed to implement the protocol. 
+Clients wishing to adopt this pattern should only need to implement UI-level changes in order to make the flow function 
+as intended, which may aid in expanding overall adoption, further enhancing the protocol's user experience benefits. 
+
+## Design Goals
+
+### Minimal effort for the user initiating the transaction
+
+The PayID Easy Checkout protocol requires a small number of points of data from the user:
+* Their PayID
+* Identifying themselves with the wallet they end up redirected to in order to approve transfer of funds to the receiver.
+
+### No server-side software specific to PayID or its protocols for servicing the transaction
+
+Since the flow of PayID Easy Checkout is predicated on using the PayID Discovery Protocol and then redirecting the 
+sender away from the site of the receiver, all of the flow can be instrumented in the browser and doesn't require server-side resources. 
+
+Apart from a PayID Discovery compliant PayID Server, The PayID Easy Checkout Protocol does not require server-side 
+software to be run by either the sender or receiver for a transaction. The PayID server is capable of providing details 
+of where to send the user via the PayID Discovery Protocol. Assuming the wallet used by the sender has implemented 
+support in their UI for the PayID Easy Checkout Protocol, the sender can be redirected to their wallet within the 
+browser to complete their transaction.
+
+# Example Usage
+This section shows the canonical PayID Easy Checkout flow between a hypothetical merchant and customer. The merchant
+accepts payments at the PayID pay$merchant.com, and the customer has the PayID alice$wallet.com.
+
+## PayID Easy Checkout Initiation
+In this example, the customer might place some items in their online cart on the merchant's website, then choose
+to checkout.  The merchant would then render a form asking for the customer's PayID, as well as a "Checkout with PayID"
+button.  Once the user inputs their PayID alice$wallet.com and clicked the "Checkout with PayID" button, the merchant
+site begins the PayID Easy Checkout flow.
+
+## PayID Easy Checkout Wallet Discovery
+The merchant site would resolve the customer's PayID to a host as defined in [PAYID-URI][], in this case resolving
+alice$wallet.com to https://wallet.com. The merchant site would then perform PayID Discovery as defined in
```

Comment:
```suggestion
alice$wallet.com to https://wallet.com. The merchant site would then continue PayID Discovery
```

Reply:
See above.  Having trouble drawing the line between what is defined in PayID URI and PayID Discovery.  Resolving the PayID URI to a host seems to be part of PayID URI, and PayID Discovery is something you can do after you have done that?

Reply:
Actually, nvm. this makes sense to me now